### PR TITLE
Adding Partagas icon

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
partagas

**Issue:** #6265 

Having an error with this one

`icons/partagas.svg
  !!! ineffective-segments ReferenceError: chain is not defined
                               at /Users/jsoria/Downloads/simple-icons/.svglintrc.js:320:21
                               at Array.forEach (<anonymous>)
                               at module.exports.rules.custom (/Users/jsoria/Downloads/simple-icons/.svglintrc.js:293:22)
                               at CustomRule (/Users/jsoria/Downloads/simple-icons/node_modules/svglint/src/rules/custom.js:33
                           :20)
                               at /Users/jsoria/Downloads/simple-icons/node_modules/svglint/src/lib/linting.js:108:33`

I don't know what this error is trying to say. The linter does not give a lot of feedback to work with here.

Also, I have a question regarding the title / name: This brand is called Partagás (notice the accent). I accented the simple-icons entry and the title in the SVG, but not the title of the SVG file. Is this correct?

### CODE REVIEW

**Comment by service-paradis** [comment]
Maybe @mondeja can take a look at this error.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9896,6 +9901,11 @@
             "hex": "3F6184",
             "source": "https://github.com/wallabag/logo/blob/f670395da2d85c3bbcb8dcfa8d2a339d8af5abb0/_default/icon/svg/logo-icon-black-no-bg.svg"
         },
+        {
+            "title": "Walmart",
+            "hex": "0071CE",
+            "source": "https://cdn.corporate.walmart.com/12/9d/06ecf3584e91acccc37162f09714/logowhite.svg"
+        },
```
Comment (service-paradis) [inline _data/simple-icons.json:9908]: Can you revert walmart changes from this PR?

Comment (jorgeamadosoria) [reply _data/simple-icons.json:9908]: ups. Yes, of course

**Comment by jorgeamadosoria** [comment]
Closed after discussion on #6265

**Pull request closed without merging**